Fixes #35732 | update port description & fix example (#36105)
mdn/content: files/en-us/web/api/url/port/index.md

{{ApiRef("URL API")}} {{AvailableInWorkers}}

> The **`port`** property of the {{domxref("URL")}} interface is a string containing the port number of the URL.

Reviewer: ```suggestion
The **`port`** property of the {{domxref("URL")}} interface is a string containing the port number of the URL, or the empty string if the port is the default for the protocol.
```

> [!NOTE]

> > If an input string passed to the {{domxref("URL.URL", "URL()")}} constructor doesn't contain an explicit port number (e.g., `https://localhost`) or contains a port number that's the default port number corresponding to the protocol part of the input string (e.g., `https://localhost:443`), then in the {{domxref("URL")}} object the constructor returns, the value of the `port` property will be the empty string: `''`.

Reviewer: ```suggestion
> If the input string passed to the {{domxref("URL.URL", "URL()")}} constructor doesn't contain an explicit port number (e.g., `https://localhost`) or contains a port number that's the default port number corresponding to the protocol part of the input string (e.g., `https://localhost:443`), or the `port` has later been set to the default port, then the value of the `port` property will be the empty string: `''`.
```
